cli/main.rs

```diff
@@ -496,10 +496,10 @@ async fn eval_command(
       MediaType::TypeScript
     } else if ext.as_str() == "tsx" {
       MediaType::TSX
-    } else if ext.as_str() == "js" {
-      MediaType::JavaScript
-    } else {
```

Comment:
This indeed seems fishy to me, it should check for `jsx` extension instead and default to `MediaType::JavaScript` in other cases

Reply:
Only the JavaScript part in the cache was not recognized, so I deleted it, but there is a problem in the test....